src/diffusers/utils/torch_utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+PyTorch utilities: Utilities related to PyTorch
+"""
+from typing import List, Optional, Tuple, Union
+
+from . import logging
+from .import_utils import is_torch_available
+
+
+if is_torch_available():
+    import torch
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+
+def torch_randn(
```

Comment:
My only concern here is that `torch_randn` is easy to confuse (both visually and inadvertently while typing) with `torch.randn`. Would it make sense to make the name slightly more different? Can't think of anything great though, `diffusers_randn` feels kind of ugly.

Reply:
Maybe just `randn_tensor`?